### PR TITLE
Uses target="_blank" for links in install.php

### PR DESCRIPTION
**Pull request by arteymix**
This PR depends on #54 to be merged as the composer check have been moved to mandatory tests.

### CODE REVIEW

**Comment by lenton** [comment]
Do you need to rebase or would merging the same commit not show up twice?

File: public/install.php [hunk 1]
```diff
@@ -186,7 +194,7 @@
 			<?php if (extension_loaded('http')): ?>
 				<td class="pass">Pass</td>
 			<?php else: ?>
-				<td class="fail">Kohana can use the <a href="http://php.net/http">http</a> extension for the Request_Client_External class.</td>
+				<td class="fail">Kohana can use the <a target="_blank" target="_blank" href="http://php.net/http">http</a> extension for the Request_Client_External class.</td>
```
Comment (lenton) [inline public/install.php]: There's two here.

**Comment by arteymix** [comment]
I rebased it, GitHub generates merge commit that breaks the fast-forward flow.

**Comment by lenton** [comment]
Thanks for your effort!

**Comment by arteymix** [comment]
@lenton I use Kohana as a basis for most of my web development projects, so it's nothing, really.

I would say thanks to you for providing such an awesome tool. :+1:

**Pull request merged**